packages/wdio-types/src/Capabilities.ts

```diff
@@ -370,6 +370,13 @@ export interface WebdriverIOCapabilities {
      * the WebDriver protocol only.
      */
     'wdio:enforceWebDriverClassic'?: boolean
+
+    /**
+     * Regular expression patterns to use when masking user input in commands having a text parameter
+     * Example: `wdio:maskingPatterns: [/@mask@/i.toString()]`
+     * Note: Because RegExp are not serializable, you need to convert them to string
```

Comment:
Can we add some real word examples here to make the description more expressive? Like how would someone mask environment variables?

Reply:
🤔 Targeting `environment variables` was not totally within my scope. As a first step, I wanted to target the text of commands. We could increase the scope, but I was expected to test a first iteration and adjust it later. What do you think?